source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -279,7 +279,8 @@ void RawHttpClientImpl::check(RequestCallbacks& callbacks,
     auto options = Http::AsyncClient::RequestOptions()
                        .setTimeout(config_->timeout())
                        .setParentSpan(parent_span)
-                       .setChildSpanName(config_->tracingName());
+                       .setChildSpanName(config_->tracingName())
+                       .setSampled(false);
```

Comment:
It would be better to add new interface to get sampled status from parent span and then just use `.setSampled(parent_span.sampled())` to achieve your original target.

Reply:
Yep yep, that was my original idea too. But I hit some bumps when trying to implement `getSampled` in the `Span` implementations in tracers, and I thought ookay first keep it simple, and we'll see from there 😅

Reply:
For example in the opentracing `Span` interface, I couldn't find a way to retrieve the sampled tag...